Makefile

```diff
@@ -49,3 +49,25 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+
+.PHONY: bench
+bench:
+	rm -rf bench
+	mkdir bench
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=${DURATION} -count=${COUNT} > bench/bench.txt
```

Comment:
I would duplicate output instead of redirect:
`go test -bench=. -benchmem -benchtime=3s -count=5 2>&1 | tee bench/b.txt`
Otherwise, command without any output for about several minutes looks hangs.

For other commands applicable too.

Reply:
Fixed.